docs/src/app/new/(content)/components/dialog/page.mdx

```diff
@@ -1,6 +1,7 @@
 # Dialog
 
 <Subtitle>A popup that opens on top of the entire page.</Subtitle>
+<meta name="description" content="A high-quality unstyled dialog component for React." />
```

Comment:
Point of interest @colmtuite: instead of using the Subtitle above, I added a separate `<meta>` tag for SEO and link descriptions on socials.

Does this approach and the copy feel right?

Reply:
"for React" bothers me slightly
"A high-quality, unstyled React dialog component" maybe?

Reply:
@vladmoroz Yep, I like the approach and copy.

Reply:
Can we use a custom wrapper component over `meta`? It'll make it staightforward to implement https://github.com/mui/base-ui/issues/612 in the future.

Reply:
> "A high-quality, unstyled React dialog component" maybe?"

Sure

Reply:
@michaldudak sure, I added the 170 char limit right away